### ISSUE TITLE
Terminal is not working

### ISSUE THREAD

**Issue by yaacovdesign** [Microsoft/vscode#31340]
- VSCode Version: Code - Insiders 1.15.0-insider (81e812c06540d1bd2636cf70ff7d0edd1bb0e866, 2017-07-21T13:15:10.775Z)
- OS Version: Windows_NT x64 10.0.15063
- Extensions: 

Extension|Author (truncated)|Version
---|---|---
vscode-twig-pack|baj|1.0.0
jshint|dba|0.10.15
vscode-eslint|dba|1.2.11
vscode-babel-coloring|dza|0.0.4
EditorConfig|Edi|0.9.4
tslint|eg2|0.17.0
Angular2|joh|2.4.0
debugger-for-chrome|msj|3.1.6
vscode-docker|Pet|0.0.16
vscode-gulp|tan|0.0.4
vscode-react-native|vsm|0.3.2

(1 theme extensions excluded)
---

Steps to Reproduce:

1. Just open the terminal inside the editor

It shows up "undefined" option on combo box that I select the terminal panel.

Reproduces without extensions: Yes

**Comment by Tyriar** [comment]
@yaacovdesign can you check the devtools console for any errors? (Help > Toggle Developer Tools)

**Comment by yaacovdesign** [comment]
@Tyriar The devtools don't report any error. The integrated terminal doesn't work and the dropdown shows the label undefined.

**Comment by rianadon** [comment]
I also did have an issue like this and the problem for me was that I had `terminal.integrated.shell.windows` set to `C:\WINDOWS\Sysnative\\cmd.exe` with 32 bit VS Code, but after upgrading to 64 bit, the dropdown showed "undefined".

I fixed this by switching the terminal to `C:\WINDOWS\System32\\cmd.exe`. I'm not sure if this is the cause of @yaacovdesign's problem though.

If this is the cause then the larger problem causing this is that any invalid terminal path causes no errors in the developer console or anywhere else - the only sign is that the terminal doesn't launch and the label is "undefined".
